README.md

```diff
@@ -25,7 +25,37 @@ Read
 [_PDD in Action_](http://www.yegor256.com/2017/04/05/pdd-in-action.html)
 and watch [this webinar](https://www.youtube.com/watch?v=nsYGC2aUwfQ).
 
-You should have [Ruby installed](https://www.ruby-lang.org/en/documentation/installation/).
+## Installation
+<details><summary>
+
+### Prerequisites: `Ruby installed` and `Libmagic`
+</summary>
+<details><summary>
+
+#### Ruby version 2.7+
+</summary>
+[Ruby installation instruction](https://www.ruby-lang.org/en/documentation/installation/)
+</details>
+<details>
+<summary>
+
+#### Libmagic library
+</summary>
+
+**For \*nix**
+```bash
+$ apt install libmagic-dev
```

Comment:
Strictly speaking, this applies only to Debian/Ubuntu/ALT Linux because other distros don't use `apt`. Perhaps, it's better to explicitly specify these distros?

Reply:
Changed